automation-tools/README.md

```diff
@@ -0,0 +1,6 @@
+# Automation tools
+
```

Comment:
A brief description here would be nice.

Reply:
Thant's WIP..-;) Yes, the whole text needs some care.